packages/walletconnect/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@celo-tools/walletconnect",
```

Comment:
I have some thoughts about what we should name this package. Part of me wants to mimic the other wallet connect wrapper name. So something like,
"@celo/wallet-walletconnect-v1"
Especially since we plan to move the other wrapper into here alongside this one. It seems like it would mesh better if they had the same sort of name.
What are your thoughts?

Reply:
I'm okay with that :)